### DISCUSSION TITLE
AttributeError: module 'pdfminer' has no attribute 'pdftypes'

### DISCUSSION THREAD

**Discussion by GemRK** [jsvine/pdfplumber#367] [Q&A]
I have just written the basic code of pdfplumber and have installed both pdfminer and pdfplumber. I am unable to figure out the error.

Code:

import pdfplumber
with pdfplumber.open('Sa Ara.pdf', passsword ="") as pdf:
    page = pdf.pages[0]
    text = page.extract_text()

Error:

  File "C:\Users\Documents\plumber.py", line 7, in <module>
    import pdfplumber

  File "C:\ProgramData\Anaconda3\lib\site-packages\pdfplumber\__init__.py", line 16, in <module>
    pdfminer.pdftypes.STRICT = False

AttributeError: module 'pdfminer' has no attribute 'pdftypes'

**Comment by samkit-jain** [comment]
Hi @GemRK . Thanks for trying out `pdfplumber`. Could you please provide the following
1. How did you install `pdfplumber` and `pdfminer.six`?
2. What version of `pdfplumber` and `pdfminer.six` is installed?
